lib/widgets/home.dart

```diff
@@ -2,96 +2,532 @@ import 'package:flutter/material.dart';
-        data: ElevatedButtonThemeData(style: ButtonStyle(
-          backgroundColor: WidgetStatePropertyAll(colorScheme.secondaryContainer),
-          foregroundColor: WidgetStatePropertyAll(colorScheme.onSecondaryContainer))),
-        child: Center(
-          child: Column(mainAxisAlignment: MainAxisAlignment.center, children: [
-            DefaultTextStyle.merge(
-              textAlign: TextAlign.center,
-              style: const TextStyle(fontSize: 18),
-              child: Column(children: [
-                Text.rich(TextSpan(
-                  text: 'Connected to: ',
-                  children: [bold(store.realmUrl.toString())])),
-              ])),
-            const SizedBox(height: 16),
-            ElevatedButton(
-              onPressed: () => Navigator.push(context,
-                MessageListPage.buildRoute(context: context,
-                  narrow: const CombinedFeedNarrow())),
-              child: Text(zulipLocalizations.combinedFeedPageTitle)),
-            const SizedBox(height: 16),
-            ElevatedButton(
-              onPressed: () => Navigator.push(context,
-                MessageListPage.buildRoute(context: context,
-                  narrow: const MentionsNarrow())),
-              child: Text(zulipLocalizations.mentionsPageTitle)),
-            const SizedBox(height: 16),
-            ElevatedButton(
-              onPressed: () => Navigator.push(context,
-                MessageListPage.buildRoute(context: context,
-                  narrow: const StarredMessagesNarrow())),
-              child: Text(zulipLocalizations.starredMessagesPageTitle)),
-            const SizedBox(height: 16),
-            ElevatedButton(
-              onPressed: () => Navigator.push(context,
-                InboxPage.buildRoute(context: context)),
-              child: const Text("Inbox")), // TODO(i18n)
-            const SizedBox(height: 16),
-            ElevatedButton(
-              onPressed: () => Navigator.push(context,
-                SubscriptionListPage.buildRoute(context: context)),
-              child: const Text("Subscribed channels")),
-            const SizedBox(height: 16),
-            ElevatedButton(
-              onPressed: () => Navigator.push(context,
-                RecentDmConversationsPage.buildRoute(context: context)),
-              child: Text(zulipLocalizations.recentDmConversationsPageTitle)),
-            if (testStreamId != null) ...[
-              const SizedBox(height: 16),
-              ElevatedButton(
-                onPressed: () => Navigator.push(context,
-                  MessageListPage.buildRoute(context: context,
-                    narrow: ChannelNarrow(testStreamId!))),
-                child: const Text("#test here")), // scaffolding hack, see above
-            ],
+      body: Column(
+        crossAxisAlignment: CrossAxisAlignment.end,
+        children: [
+          const Expanded(child: LoadingPlaceholder()),
+          TextButton(
```

Comment:
This needs to use SafeArea to stay out of the insets. Here's how it looks on my phone:
![image](https://github.com/user-attachments/assets/aa4231f6-2533-4d87-b10e-79faae13826d)

Reply:
Probably cleanest is to make this a floating action button:
```dart
    return Scaffold(
      body: const LoadingPlaceholder(),
      floatingActionButton: TextButton(
        onPressed: () => Navigator.push(context,
          MaterialWidgetRoute(page: const ChooseAccountPage())),
        child: Text(zulipLocalizations.chooseAccountPageTitle)),
    );
```

That takes care of applying SafeArea; and it also means the layout of LoadingPlaceholder and centering of the circular progress indicator doesn't get distorted by the presence of the button.

Reply:
In this revision I also added the `AppBar` back because `HomePage` has gained one in the previous update.